client/layout/masterbar/style.scss

```diff
@@ -38,7 +43,7 @@ $autobar-height: 20px;
 	display: flex;
 		align-items: center;
 	position: relative;
-	color: $white;
+	@include theme( color, --masterbar-color );
```

Comment:
Sorry yes, I was confused that when we decided to keep IE11 as being progressive enhancement (thus, no themes) we solved this issue. As you said, it's not the case.

...

I gave a thought about it, and I narrowed down why this feels odd to be done in this way: it's the same scope for which `autoprefixer` exists, right? The scenario is that we don't want to care about writing any different SCSS/CSS for backward compatibility, we let the processor to deal with it.

So, I'd expect that the above syntax to be just:

`color: var( --some-theme-color, $wordpress_blue );`

And that this would be expanded by `PostCSS` or similar processor to:

```
color: #0087be;
color: var( --some-theme-color, 	#0087be );
```

...probably through an intermediate step that replaced the SCSS variable.

Can you look into this?